<div id=toc></div>

# Table of Contents

- [q-bio.GN](#q-bio.GN) [Total: 1]
- [q-bio.QM](#q-bio.QM) [Total: 7]
- [math.PR](#math.PR) [Total: 1]
- [cs.CV](#cs.CV) [Total: 1]
- [cs.LG](#cs.LG) [Total: 2]


<div id='q-bio.GN'></div>

# q-bio.GN [[Back]](#toc)

### [1] [Deep Generative Models for Discrete Genotype Simulation](https://arxiv.org/abs/2508.09212)
*Sihan Xie,Thierry Tribout,Didier Boichard,Blaise Hanczar,Julien Chiquet,Eric Barrey*

Main category: q-bio.GN

TL;DR: 该研究探讨了生成离散基因型数据的深度生成模型，比较了VAEs、扩散模型和GANs的性能，并提供了实用指南。


<details>
  <summary>Details</summary>
Motivation: 解决基因型数据生成的挑战，同时保护隐私并提高数据可访问性。

Method: 开发并评估了VAEs、扩散模型和GANs，针对离散基因型数据进行了适应性调整。

Result: 模型能有效捕捉遗传模式并保留基因型-表型关联。

Conclusion: 研究为基因型模拟提供了全面的模型比较和实用指南，代码已公开。

Abstract: Deep generative models open new avenues for simulating realistic genomic data
while preserving privacy and addressing data accessibility constraints. While
previous studies have primarily focused on generating gene expression or
haplotype data, this study explores generating genotype data in both
unconditioned and phenotype-conditioned settings, which is inherently more
challenging due to the discrete nature of genotype data. In this work, we
developed and evaluated commonly used generative models, including Variational
Autoencoders (VAEs), Diffusion Models, and Generative Adversarial Networks
(GANs), and proposed adaptation tailored to discrete genotype data. We
conducted extensive experiments on large-scale datasets, including all
chromosomes from cow and multiple chromosomes from human. Model performance was
assessed using a well-established set of metrics drawn from both deep learning
and quantitative genetics literature. Our results show that these models can
effectively capture genetic patterns and preserve genotype-phenotype
association. Our findings provide a comprehensive comparison of these models
and offer practical guidelines for future research in genotype simulation. We
have made our code publicly available at
https://github.com/SihanXXX/DiscreteGenoGen.

</details>


<div id='q-bio.QM'></div>

# q-bio.QM [[Back]](#toc)

### [2] [Real-time deep learning phase imaging flow cytometer reveals blood cell aggregate biomarkers for haematology diagnostics](https://arxiv.org/abs/2508.09215)
*Kerem Delikoyun,Qianyu Chen,Liu Wei,Si Ko Myo,Johannes Krell,Martin Schlegel,Win Sen Kuan,John Tshon Yit Soong,Gerhard Schneider,Clarissa Prazeres da Costa,Percy A. Knolle,Laurent Renia,Matthew Edward Cove,Hwee Kuan Lee,Klaus Diepold,Oliver Hayden*

Main category: q-bio.QM

TL;DR: RT-HAD是一个基于深度学习的框架，用于实时处理血液细胞图像数据，解决传统方法在检测细胞聚集体时的不足。


<details>
  <summary>Details</summary>
Motivation: 传统血液学方法无法有效检测细胞聚集体，且现有技术面临数据存储和处理难题。RT-HAD旨在提供一种快速、准确的解决方案。

Method: 结合物理一致的全息重建和检测技术，将每个血细胞表示为图结构，利用深度学习识别聚集体。

Result: RT-HAD能在1.5分钟内处理30GB数据，血小板聚集体检测错误率为8.9%，符合临床标准。

Conclusion: RT-HAD解决了即时诊断中的大数据挑战，显著提升了无标记功能诊断的可行性。

Abstract: While analysing rare blood cell aggregates remains challenging in automated
haematology, they could markedly advance label-free functional diagnostics.
Conventional flow cytometers efficiently perform cell counting with leukocyte
differentials but fail to identify aggregates with flagged results, requiring
manual reviews. Quantitative phase imaging flow cytometry captures detailed
aggregate morphologies, but clinical use is hampered by massive data storage
and offline processing. Incorporating hidden biomarkers into routine
haematology panels would significantly improve diagnostics without flagged
results. We present RT-HAD, an end-to-end deep learning-based image and data
processing framework for off-axis digital holographic microscopy (DHM), which
combines physics-consistent holographic reconstruction and detection,
representing each blood cell in a graph to recognize aggregates. RT-HAD
processes >30 GB of image data on-the-fly with turnaround time of <1.5 min and
error rate of 8.9% in platelet aggregate detection, which matches acceptable
laboratory error rates of haematology biomarkers and solves the big data
challenge for point-of-care diagnostics.

</details>


### [3] [Exploring Molecular Odor Taxonomies for Structure-based Odor Predictions using Machine Learning](https://arxiv.org/abs/2508.09217)
*Akshay Sajan,Stijn Sluis,Reza Haydarlou,Sanne Abeln,Pasquale Lisena,Raphael Troncy,Caro Verbeek,Inger Leemans,Halima Mouhib*

Main category: q-bio.QM

TL;DR: 论文提出通过专家分类法和数据驱动分类法改进机器学习模型预测气味的能力，并展示了两种方法的有效性。


<details>
  <summary>Details</summary>
Motivation: 解决从分子结构预测气味的挑战，尤其是对气味空间和结构-气味关系的有限理解。

Method: 使用专家分类法（基于语义和感知相似性）和数据驱动分类法（基于气味描述符的共现模式聚类）改进机器学习模型。

Result: 两种分类法均提升了预测性能，优于随机分组，并通过深入错误分析揭示了气味-结构关系的复杂性。

Conclusion: 两种分类法为气味研究提供了新工具，数据驱动分类法有助于验证专家分类法，推动气味分子空间的探索。

Abstract: One of the key challenges to predict odor from molecular structure is
unarguably our limited understanding of the odor space and the complexity of
the underlying structure-odor relationships. Here, we show that the predictive
performance of machine learning models for structure-based odor predictions can
be improved using both, an expert and a data-driven odor taxonomy. The expert
taxonomy is based on semantic and perceptual similarities, while the
data-driven taxonomy is based on clustering co-occurrence patterns of odor
descriptors directly from the prepared dataset. Both taxonomies improve the
predictions of different machine learning models and outperform random
groupings of descriptors that do not reflect existing relations between odor
descriptors. We assess the quality of both taxonomies through their predictive
performance across different odor classes and perform an in-depth error
analysis highlighting the complexity of odor-structure relationships and
identifying potential inconsistencies within the taxonomies by showcasing pear
odorants used in perfumery. The data-driven taxonomy allows us to critically
evaluate our expert taxonomy and better understand the molecular odor space.
Both taxonomies as well as a full dataset are made available to the community,
providing a stepping stone for a future community-driven exploration of the
molecular basis of smell. In addition, we provide a detailed multi-layer expert
taxonomy including a total of 777 different descriptors from the Pyrfume
repository.

</details>


### [4] [Cross-BCI, A Cross-BCI-Paradigm Classifica-tion Model Towards Universal BCI Applications](https://arxiv.org/abs/2508.09242)
*Gaojie Zhou,Junhua Li*

Main category: q-bio.QM

TL;DR: 提出了一种轻量级、统一的跨BCI范式解码模型，显著优于现有模型，为低成本通用应用奠定基础。


<details>
  <summary>Details</summary>
Motivation: 现有BCI分类模型通常针对单一范式设计，重新开发成本高，且需要轻量级模型以适应便携设备。

Method: 模型结合时空卷积、多尺度局部特征选择和多维全局特征提取模块，生成高级特征表示。

Result: 在三种经典BCI范式上，模型准确率达88.39%，显著优于对比模型。

Conclusion: 该模型为跨BCI范式分类提供了可行方案，推动了低成本通用解码系统的发展。

Abstract: Classification models used in brain-computer interface (BCI) are usually
designed for a single BCI paradigm. This requires the redevelopment of the
model when applying it to a new BCI paradigm, resulting in repeated costs and
effort. Moreover, less complex deep learning models are desired for practical
usage, as well as for deployment on portable devices. In or-der to fill the
above gaps, we, in this study, proposed a light-weight and unified decoding
model for cross-BCI-paradigm classification. The proposed model starts with a
tempo-spatial convolution. It is followed by a multi-scale local feature
selec-tion module, aiming to extract local features shared across BCI paradigms
and generate weighted features. Finally, a mul-ti-dimensional global feature
extraction module is designed, in which multi-dimensional global features are
extracted from the weighted features and fused with the weighted features to
form high-level feature representations associated with BCI para-digms. The
results, evaluated on a mixture of three classical BCI paradigms (i.e., MI,
SSVEP, and P300), demon-strate that the proposed model achieves 88.39%, 82.36%,
80.01%, and 0.8092 for accuracy, macro-precision, mac-ro-recall, and
macro-F1-score, respectively, significantly out-performing the compared models.
This study pro-vides a feasible solution for cross-BCI-paradigm
classifica-tion. It lays a technological foundation for de-veloping a new
generation of unified decoding systems, paving the way for low-cost and
universal practical applications.

</details>


### [5] [Predictive Significance of CD276/B7-H3 Expression in Baseline Biopsies of Advanced Prostate Carcinoma](https://arxiv.org/abs/2508.09373)
*Adam Yusuf,Paramahansa Pramanik*

Main category: q-bio.QM

TL;DR: 研究通过测量CD276/B7-H3蛋白水平，发现其高表达与前列腺癌转移、生存期缩短及肿瘤侵袭性相关，可作为早期高风险患者识别和治疗靶点。


<details>
  <summary>Details</summary>
Motivation: 前列腺癌诊断时病情差异大，长期预后预测困难，需寻找可靠生物标志物。

Method: 分析248名男性诊断组织样本中的B7-H3水平，比较局限性与转移性癌症患者。

Result: 转移患者B7-H3水平更高，高表达与生存期短、PSA高及Gleason评分高相关。

Conclusion: B7-H3可作为早期高风险识别标志物，并为晚期前列腺癌治疗提供新靶点。

Abstract: At the time of diagnosis, prostate cancer can appear deceptively mild or
already display signs of widespread disease. Predicting long-term outcomes is
often uncertain. This research focused on measuring CD276/B7-H3, an immune
checkpoint protein linked to tumor development, in diagnostic tissue samples
from 248 men. Participants included both those with cancer confined to the
prostate and those with confirmed metastases. Analysis showed that patients
with metastatic disease were more likely to exhibit increased B7-H3 levels.
Strong expression of this marker was associated with shorter survival times and
was observed alongside higher PSA concentrations and greater tumor
aggressiveness based on Gleason grading. These trends remained consistent even
when other prognostic factors were taken into account. The results suggest that
assessing B7-H3 during the initial biopsy could help clinicians identify
high-risk patients earlier. This marker may also represent a new target for
treatment strategies in advanced prostate cancer.

</details>


### [6] [CAKL: Commutative algebra k-mer learning of genomics](https://arxiv.org/abs/2508.09406)
*Faisal Suwayyid,Yuta Hozumi,Hongsong Feng,Mushal Zia,JunJie Wee,Guo-Wei Wei*

Main category: q-bio.QM

TL;DR: 论文提出了一种新的非线性代数框架CAKL，用于基因组序列分析，结合了交换代数、代数拓扑、组合数学和机器学习，在多个任务中表现优于现有方法。


<details>
  <summary>Details</summary>
Motivation: 尽管有多种序列分析模型可用，但比较基因组分析在基因组学、遗传学和系统发育学中仍具挑战性。交换代数作为代数几何和数论的基础工具，很少用于数据科学和生物学。

Method: 引入交换代数k-mer学习（CAKL）框架，结合交换代数、代数拓扑、组合数学和机器学习，建立新的数学范式。

Result: 在三个任务（遗传变异识别、系统发育树分析和病毒基因组分类）中，CAKL在11个数据集上优于五种最先进的序列分析方法，尤其在病毒分类中表现突出，且具有可扩展性和鲁棒性。

Conclusion: CAKL开创了交换代数在数据分析和学习中的新纪元。

Abstract: Despite the availability of various sequence analysis models, comparative
genomic analysis remains a challenge in genomics, genetics, and phylogenetics.
Commutative algebra, a fundamental tool in algebraic geometry and number
theory, has rarely been used in data and biological sciences. In this study, we
introduce commutative algebra k-mer learning (CAKL) as the first-ever nonlinear
algebraic framework for analyzing genomic sequences. CAKL bridges between
commutative algebra, algebraic topology, combinatorics, and machine learning to
establish a new mathematical paradigm for comparative genomic analysis. We
evaluate its effectiveness on three tasks -- genetic variant identification,
phylogenetic tree analysis, and viral genome classification -- typically
requiring alignment-based, alignment-free, and machine-learning approaches,
respectively. Across eleven datasets, CAKL outperforms five state-of-the-art
sequence analysis methods, particularly in viral classification, and maintains
stable predictive accuracy as dataset size increases, underscoring its
scalability and robustness. This work ushers in a new era in commutative
algebraic data analysis and learning.

</details>


### [7] [Quest for a clinically relevant medical image segmentation metric: the definition and implementation of Medical Similarity Index](https://arxiv.org/abs/2508.09722)
*Szuzina Fazekas,Bettina Katalin Budai,Viktor Bérczi,Pál Maurovich-Horvat,Zsolt Vizi*

Main category: q-bio.QM

TL;DR: 提出了一种新的医学图像分割评估指标，结合了临床相关性，并提供了开源处理流程。


<details>
  <summary>Details</summary>
Motivation: 传统分割评估指标仅基于几何特性，无法适应多样化临床应用，需开发更灵活的评估方法。

Method: 定义双向局部距离，配对测试与参考轮廓点，计算欧氏距离并评分，最终得出医学相似性指数。

Result: 开发了易用的Python处理流程，支持多切片图像和多掩码处理，并在前列腺分割中验证了适应性。

Conclusion: 新指标和开源工具可方便地评估医学图像分割的临床相关性。

Abstract: Background: In the field of radiology and radiotherapy, accurate delineation
of tissues and organs plays a crucial role in both diagnostics and
therapeutics. While the gold standard remains expert-driven manual
segmentation, many automatic segmentation methods are emerging. The evaluation
of these methods primarily relies on traditional metrics that only incorporate
geometrical properties and fail to adapt to various applications.
  Aims: This study aims to develop and implement a clinically relevant
segmentation metric that can be adapted for use in various medical imaging
applications.
  Methods: Bidirectional local distance was defined, and the points of the test
contour were paired with points of the reference contour. After correcting for
the distance between the test and reference center of mass, Euclidean distance
was calculated between the paired points, and a score was given to each test
point. The overall medical similarity index was calculated as the average score
across all the test points. For demonstration, we used myoma and prostate
datasets; nnUNet neural networks were trained for segmentation.
  Results: An easy-to-use, sustainable image processing pipeline was created
using Python. The code is available in a public GitHub repository along with
Google Colaboratory notebooks. The algorithm can handle multislice images with
multiple masks per slice. Mask splitting algorithm is also provided that can
separate the concave masks. We demonstrate the adaptability with prostate
segmentation evaluation.
  Conclusions: A novel segmentation evaluation metric was implemented, and an
open-access image processing pipeline was also provided, which can be easily
used for automatic measurement of clinical relevance of medical image
segmentation.}

</details>


### [8] [NEUBORN: The Neurodevelopmental Evolution framework Using BiOmechanical RemodelliNg](https://arxiv.org/abs/2508.09757)
*Nashira Baena,Mariana da Silva,Irina Grigorescu,Aakash Saboo,Saga Masui,Jaques-Donald Tournier,Emma C. Robinson*

Main category: q-bio.QM

TL;DR: 提出了一种基于生物力学约束的个体生长轨迹学习框架，用于改进皮层发育的建模。


<details>
  <summary>Details</summary>
Motivation: 当前规范建模框架难以捕捉精细解剖细节，因其依赖群体平均参考空间。

Method: 通过分层网络架构实现生物力学约束的纵向微分同胚图像配准。

Result: 方法生成更平滑的变形和更少的负雅可比行列式，提高了生物合理性。

Conclusion: 该框架为预测脑成熟和早期识别皮层发育畸形提供了新可能。

Abstract: Understanding individual cortical development is essential for identifying
deviations linked to neurodevelopmental disorders. However, current normative
modelling frameworks struggle to capture fine-scale anatomical details due to
their reliance on modelling data within a population-average reference space.
Here, we present a novel framework for learning individual growth trajectories
from biomechanically constrained, longitudinal, diffeomorphic image
registration, implemented via a hierarchical network architecture. Trained on
neonatal MRI data from the Developing Human Connectome Project, the method
improves the biological plausibility of warps, generating growth trajectories
that better follow population-level trends while generating smoother warps,
with fewer negative Jacobians, relative to state-of-the-art baselines. The
resulting subject-specific deformations provide interpretable, biologically
grounded mappings of development. This framework opens new possibilities for
predictive modeling of brain maturation and early identification of
malformations of cortical development.

</details>


<div id='math.PR'></div>

# math.PR [[Back]](#toc)

### [9] [Stability of randomly switching stochastic reaction networks with asymptotically linear transition rates](https://arxiv.org/abs/2507.09421)
*Daniele Cappelletti,Aidan Howells,Chuang Xu*

Main category: math.PR

TL;DR: 研究了随机切换对随机反应网络稳定性的影响，给出了高、低切换速率下的稳定性条件，并探讨了中间速率下的相变现象。


<details>
  <summary>Details</summary>
Motivation: 随机反应网络在生物化学等领域广泛应用，但其参数（速率常数）在实际中可能变化，研究切换速率对稳定性的影响具有重要意义。

Method: 通过矩阵条件分析高、低切换速率下的稳定性（正递归和瞬态），并通过示例验证中间速率下的相变现象。

Result: 给出了高、低切换速率下的稳定性条件，并发现中间速率下可能存在多次相变。

Conclusion: 切换速率对随机反应网络的稳定性有显著影响，中间速率下可能出现复杂的相变行为。

Abstract: Stochastic reaction networks are mathematical models frequently used in, but
not limited to, biochemistry. These models are continuous-time Markov chains
whose transition rates depend on certain parameters called rate constants,
which despite the name may not be constant in real-world applications. In this
paper we study how random switching between different stochastic reaction
networks with asymptotically linear rate functions affects the stability of the
process. We give matrix conditions for both positive recurrence (indeed,
exponentially ergodicity) and transience (indeed, evanescence) in both the
regime with high switching rates and the regime with low switching rates. We
then make use of these conditions to provide examples of processes whose
stability behavior changes as the switching rate varies. We also explore what
happens in the middle regime where the switching rates are neither high nor low
and our theorems do not apply. Specifically, we show by examples that there can
be arbitrarily many phase transitions between exponentially ergodicity and
evanescence as the switching rate increases.

</details>


<div id='cs.CV'></div>

# cs.CV [[Back]](#toc)

### [10] [Towards Comprehensive Cellular Characterisation of H&E slides](https://arxiv.org/abs/2508.09926)
*Benjamin Adjadj,Pierre-Antoine Bannier,Guillaume Horent,Sebastien Mandela,Aurore Lyon,Kathryn Schutte,Ulysse Marteau,Valentin Gaury,Laura Dumont,Thomas Mathieu,Reda Belbahri,Benoît Schmauch,Eric Durand,Katharina Von Loga,Lucie Gillet*

Main category: cs.CV

TL;DR: HistoPLUS是一种先进的细胞分析模型，针对肿瘤微环境中的细胞检测、分割和分类问题，显著提升了性能并减少了参数需求。


<details>
  <summary>Details</summary>
Motivation: 现有方法在罕见细胞类型和跨域泛化上表现不佳，HistoPLUS旨在解决这些问题。

Method: 基于108,722个核的新数据集训练，覆盖13种细胞类型。

Result: 在外部验证中，检测质量提升5.2%，分类F1分数提升23.7%，参数减少5倍。

Conclusion: HistoPLUS显著改进了罕见细胞类型的研究，并支持更广泛的肿瘤微环境生物标志物研究。

Abstract: Cell detection, segmentation and classification are essential for analyzing
tumor microenvironments (TME) on hematoxylin and eosin (H&E) slides. Existing
methods suffer from poor performance on understudied cell types (rare or not
present in public datasets) and limited cross-domain generalization. To address
these shortcomings, we introduce HistoPLUS, a state-of-the-art model for cell
analysis, trained on a novel curated pan-cancer dataset of 108,722 nuclei
covering 13 cell types. In external validation across 4 independent cohorts,
HistoPLUS outperforms current state-of-the-art models in detection quality by
5.2% and overall F1 classification score by 23.7%, while using 5x fewer
parameters. Notably, HistoPLUS unlocks the study of 7 understudied cell types
and brings significant improvements on 8 of 13 cell types. Moreover, we show
that HistoPLUS robustly transfers to two oncology indications unseen during
training. To support broader TME biomarker research, we release the model
weights and inference code at https://github.com/owkin/histoplus/.

</details>


<div id='cs.LG'></div>

# cs.LG [[Back]](#toc)

### [11] [Presenting DiaData for Research on Type 1 Diabetes](https://arxiv.org/abs/2508.09160)
*Beyza Cinar,Maria Maleshkova*

Main category: cs.LG

TL;DR: 该论文整合了15个数据集，创建了一个包含2510名受试者的糖尿病数据库，用于研究低血糖事件，并分析了数据质量和血糖与心率的关系。


<details>
  <summary>Details</summary>
Motivation: 解决糖尿病研究中数据不足的问题，通过整合多个数据集提供大规模数据支持，以改进低血糖预测和糖尿病管理。

Method: 系统整合15个数据集，构建包含149百万次血糖测量的数据库，并提取两个子数据库（人口统计和心率数据），同时评估数据质量和进行相关性研究。

Result: 创建了均衡分布的大型数据库，发现数据不平衡和缺失值是主要挑战，并揭示了血糖与心率在低血糖前15至55分钟的相关性。

Conclusion: 整合的数据集为糖尿病研究提供了重要资源，数据质量问题和血糖与心率的关系为未来研究提供了方向。

Abstract: Type 1 diabetes (T1D) is an autoimmune disorder that leads to the destruction
of insulin-producing cells, resulting in insulin deficiency, as to why the
affected individuals depend on external insulin injections. However, insulin
can decrease blood glucose levels and can cause hypoglycemia. Hypoglycemia is a
severe event of low blood glucose levels ($\le$70 mg/dL) with dangerous side
effects of dizziness, coma, or death. Data analysis can significantly enhance
diabetes care by identifying personal patterns and trends leading to adverse
events. Especially, machine learning (ML) models can predict glucose levels and
provide early alarms. However, diabetes and hypoglycemia research is limited by
the unavailability of large datasets. Thus, this work systematically integrates
15 datasets to provide a large database of 2510 subjects with glucose
measurements recorded every 5 minutes. In total, 149 million measurements are
included, of which 4% represent values in the hypoglycemic range. Moreover, two
sub-databases are extracted. Sub-database I includes demographics, and
sub-database II includes heart rate data. The integrated dataset provides an
equal distribution of sex and different age levels. As a further contribution,
data quality is assessed, revealing that data imbalance and missing values
present a significant challenge. Moreover, a correlation study on glucose
levels and heart rate data is conducted, showing a relation between 15 and 55
minutes before hypoglycemia.

</details>


### [12] [Thermal Tracks: A Gaussian process-based framework for universal melting curve analysis enabling unconstrained hit identification in thermal proteome profiling experiments](https://arxiv.org/abs/2508.09659)
*Johannes F. Hevler,Shivam Verma,Mirat Soijtra,Carolyn R. Bertozzi*

Main category: cs.LG

TL;DR: Thermal Tracks是一个基于Python的统计框架，用于分析蛋白质热稳定性数据，克服了现有热蛋白质组分析（TPP）方法的局限性。


<details>
  <summary>Details</summary>
Motivation: 现有TPP方法假设蛋白质熔解曲线为S形，并受限于经验零分布，导致仅能检测约5%的数据。Thermal Tracks旨在解决这一问题，提供更灵活的分析方法。

Method: 使用高斯过程（GP）模型和平方指数核，灵活建模任何熔解曲线形状，并通过核先验生成无偏零分布。

Result: Thermal Tracks能够分析蛋白质组范围内的扰动（如通路抑制、基因修饰或环境压力），并适用于具有非传统熔解曲线的蛋白质（如相分离蛋白和膜蛋白）。

Conclusion: Thermal Tracks是一个免费、灵活的工具，适用于广泛的蛋白质热稳定性研究。

Abstract: Thermal Tracks is a Python-based statistical framework for analyzing protein
thermal stability data that overcomes key limitations of existing thermal
proteome profiling (TPP) work-flows. Unlike standard approaches that assume
sigmoidal melting curves and are constrained by empirical null distributions
(limiting significant hits to approximately 5 % of data), Thermal Tracks uses
Gaussian Process (GP) models with squared-exponential kernels to flexibly model
any melting curve shape while generating unbiased null distributions through
kernel priors. This framework is particularly valuable for analyzing
proteome-wide perturbations that significantly alter protein thermal stability,
such as pathway inhibitions, genetic modifications, or environmental stresses,
where conventional TPP methods may miss biologically relevant changes due to
their statistical constraints. Furthermore, Thermal Tracks excels at analyzing
proteins with un-conventional melting profiles, including phase-separating
proteins and membrane proteins, which often exhibit complex, non-sigmoidal
thermal stability behaviors. Thermal Tracks is freely available from GitHub and
is implemented in Python, providing an accessible and flexible tool for
proteome-wide thermal profiling studies.

</details>
